accession/utils.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import Count
+
+from accession import models
+
+
+class ModelNotFound(Exception):
+    """Exception for when the model is not in the model_field_map."""
+    pass
+
+
+def find_duplicates(model_str):
+    """Finds all table entries that have the same value in a certain field.
+
+    This function takes a model string and returns a list of
+    lists containing the count and field name for all found duplicates.
+    For example, if there was a model 'Dogs' with a field 'name', and two of
+    the entries shared the same name 'Ralf', then the returned results would
+    be the following:
+        >>>find_duplicates(Dogs, 'name')
+        [[2, 'Ralf']]
+    """
+
+    model_field_map = {
+        'accession': [models.Accession, 'accession_number'],
```

Comment:
This looks great!  One suggestion though,

It might be a good idea use a tuple here instead of a list. Tuples are immutable where lists are not. In the end, I don't think it is a huge deal. Something to consider though.